Fix the comments w.r.t. configuration file usage with the -T option.
This change has been submitted upstream.

Fix the references to networks(4) - should be in networks(5).
This change will be sent upstream.

--- snort-2.9.18/snort.8
+++ snort-2.9.18/snort.8
@@ -341,8 +341,7 @@ command line switches and rules files th
 indicating that everything is ready to proceed.  This is a good
 switch to use if daemon mode is going to be used, it verifies that
 the Snort configuration that is about to be used is valid and won't fail at
-run time. Note, Snort looks for either /etc/snort.conf or ./snort.conf.
-If your config lives elsewhere, use the -c option to specify a valid
+run time. Note that you will need to use the -c option to specify a valid
 .I config-file.
 .IP "-u user"
 Change the user/UID Snort runs under to
@@ -605,7 +604,7 @@ which can be used with either names or n
 .IP "\fBdst net \fInet\fR"
 True if the IP destination address of the packet has a network
 number of \fInet\fP. \fINet\fP may be either a name from /etc/networks
-or a network number (see \fInetworks(4)\fP for details).
+or a network number (see \fInetworks(5)\fP for details).
 .IP "\fBsrc net \fInet\fR"
 True if the IP source address of the packet has a network
 number of \fInet\fP.
